### PR TITLE
More doc updates applicable to migrating data to a new instance

### PR DESCRIPTION
**Pull request by gherceg**
Followup to https://github.com/dimagi/commcare-cloud/pull/6214
##### Environments Affected

None

### CODE REVIEW

**Review by mkangia — APPROVED** [review]

File: docs/source/installation/2-manual-install.rst [hunk 1]
```diff
@@ -588,6 +588,8 @@ initially.
    ::
 
        $ commcare-cloud cluster deploy
+       $ Or if you need to deploy a specific version of CommCare HQ as opposed to the latest:
```
Comment (kaapstorm) [inline docs/source/installation/2-manual-install.rst:591]:
```suggestion

   Or if you need to deploy a specific version of CommCare HQ as opposed to the latest
   
   ::

```

**Review by kaapstorm — APPROVED** [review]
Just the one formatting suggestion (cos otherwise the instruction looks like a command for the user to type at the command prompt).

File: docs/source/installation/2-manual-install.rst (hunk 1 above)
Comment (gherceg) [reply docs/source/installation/2-manual-install.rst]: 92056e653c92d2b3fe15e311ef081a1dd6654595

File: docs/source/installation/2-manual-install.rst [hunk 2]
```diff
@@ -589,6 +589,12 @@ initially.
 
        $ commcare-cloud cluster deploy
 
+   Or if you need to deploy a specific version of CommCare HQ as opposed to the latest:
+
+   ::
+
+       $ commcare-cloud cluster deploy --commcare-rev=<commit-hash>
```
Comment (minhaminha) [inline docs/source/installation/2-manual-install.rst:596]: not blocking but might be good to clarify the specific scenarios in which you'll need to deploy with a specific version, unless they read this already knowing they have to (like if they were redirected from the project migration docs or something)

**Review by minhaminha — APPROVED** [review]

**Review by mkangia — APPROVED** [review]

File: docs/source/installation/2-manual-install.rst (hunk 2 above)
Comment (gherceg) [reply docs/source/installation/2-manual-install.rst:596]: Yeah the docs for a data migration mention that they will need to deploy from a specific commit, and we will share a commit hash with them which should also remind them that they will need to use it. I'd rather not add too much to the manual install docs about this, but I imagine there is still room for improvement.

**Pull request merged**